api/ipfsproxy/ipfsproxy_test.go

```diff
@@ -314,6 +314,108 @@ func TestIPFSProxyUnpin(t *testing.T) {
 	}
 }
 
+func TestIPFSProxyPinUpdate(t *testing.T) {
```

Comment:
This may be a stupid idea, but for the proxy tests, it may be 'beneficial' to make the same requests against an ipfs api and use the response for the comparison with the cluster proxy response.

Reply:
We can't mimic IPFS responses fully, particularly cluster error responses will need to be with cluster errors. In any case, we don't even use the IPFS mock to test this component, and rely fully on the RPC mock. It would not hurt to have end-2-end tests here. And in general, switch to using a real IPFS daemon and not the mock.